sdk/extensions/wcf/Microsoft.CoreWCF.Azure.StorageQueues/README.md

```diff
@@ -0,0 +1,152 @@
+});
+```
+
+If a message was recevied from the queue, but wasn't able to be processed, it will be placed in the dead letter queue. You can sepcify the dead letter queue name by passing it to the constructor of `AazureQueueStorageBinding`. If not specified, the default value of `default-dead-letter-queue` will be used.
+
+## Key concepts
+
+CoreWCF is an implementation of the service side features of Windows Communication Foundation (WCF) for .NET. The goal of this project is to enable migrating existing WCF services to .NET that are currently using MSMQ and wish to deploy their service to Azure, replacing MSMQ with Azure Queue Storage.
+
+More general samples of using CoreWCF can be found in the [CoreWCF samples repository][corewcf_repo]. To create a client to send messages to an Azure Storage Queue, see the To create a service to receive messages sent to an Azure Storage Queue, see the [Microsoft.WCF.Azure.StorageQueues documentation][wcf_docs]. 
+
+## Contributing
+
+See the [Storage CONTRIBUTING.md][storage_contrib] for details on building,testing, and contributing to this library.
+
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit [cla.microsoft.com][cla].
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc].
+For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
+
+<!-- LINKS -->
+[nuget]: https://www.nuget.org/
+[storage_account_docs]: https://learn.microsoft.com/azure/storage/common/storage-account-overview
+[storage_account_create_ps]: https://learn.microsoft.com/azure/storage/common/storage-account-create?tabs=azure-powershell
+[storage_account_create_cli]: https://learn.microsoft.com/azure/storage/common/storage-account-create?tabs=azure-cli
+[storage_account_create_portal]: https://learn.microsoft.com/azure/storage/common/storage-account-create?tabs=azure-portal
+[azure_cli]: https://learn.microsoft.com/cli/azure/
+[azure_sub]: https://azure.microsoft.com/free/dotnet/
+[identity]: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/identity/Azure.Identity/README.md
+[storage_ad]: https://learn.microsoft.com/azure/storage/blobs/authorize-access-azure-active-directory
+[storage_contrib]: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/CONTRIBUTING.md
+[cla]: https://opensource.microsoft.com/cla/
+[coc]: https://opensource.microsoft.com/codeofconduct/
+[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
+[coc_contact]: mailto:opencode@microsoft.com
+[corewcf_repo]: https://github.com/CoreWCF/samples/
+[wcf_docs]: <!--https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/extensions/wcf/Microsoft.WCF.Azure.StorageQueues-->
```

Comment:
```suggestion
[wcf_docs]: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/extensions/wcf/Microsoft.WCF.Azure.StorageQueues
```

Links to main should work as long as the links are part of this PR.

Reply:
They don't work. It's how I found the [documentation](https://github.com/Azure/azure-sdk/blob/main/docs/policies/README-TEMPLATE.md#link-guidelines) from when we first had links to main as the build provides a link to the documentation. It specifically says this:
> If the CI validation fails because of link verification for a link to a file or section present in your PR and not in the main but will be a valid link after the PR is merged to main, then you can put the link as a comment like this Here are some \[samples\]&lt;!--(https\://github.com/azure-sdk-for-python/blob/main/samples.json)--&gt; to look at.